mqtt/mqtt-bridge/src/bridge.rs

```diff
@@ -15,7 +15,7 @@ impl Bridge {
     }
 
     pub async fn start(&self) {
-        info!("Starting bridge...{:?}", self.connection_settings);
+        info!("Starting bridge...{:?}", self.connection_settings.name());
```

Comment:
nit: Could you please check formatting for all the code you added? It may not be too convenient to dump objects internal structure in the logs.
```suggestion
        info!("Starting bridge...{}", self.connection_settings.name());
```
This will use `Display` implementation to write data instead of `Debug` https://doc.rust-lang.org/std/fmt/